File: JAVA/Account.java
public class Account {
    //Propiedades o atributos 
    Integer id;
    String name;
    String document;
    String email;
    String password;

    // Método constructor
    public Account(String name, String document){
        this.name = name;
        this.document = document;
    }

    // Método
    void printDataAccount(){
        System.out.println("Name: " + this.name);
        System.out.println("Document: " + this.document);
    }
        

}

File: PHP/account.php
<?php
class Account{
    public $id;
    public $name;
    public $document; 
    public $email;
    public $password; 

    // Constructor
    public function __construct($name, $document){
        $this->name = $name;
        $this->document = $document;
    }
    
    public function printDataAccount(){
        echo "<br>";
        echo "Name: " . $this->name . " Document: " . $this->document;
    }
}
?>